docs/configuration.md

```diff
@@ -16,6 +16,13 @@ gotify/server looks in the following paths for config files
 **Note**: When strings contain reserved yml characters then they need to be escaped.
 [A list of reserved characters and how to escape them.](https://stackoverflow.com/a/22235064/4244993)
 
+**Note**: Since the config file contains sensitive data such as the
+administrator's password, it can be protected with:
```

Comment:
Per default gotify doesn't require configuration, so this file may not exist. Could you add here that this step is only necessary if gotify is configured via a config file in /etc/gotify.

Reply:
Added text regarding this. Do you want me to squash the commits?